server/test/server.spec.ts

```diff
@@ -0,0 +1,27 @@
+// 
+// Note: This example test is leveraging the Mocha test framework.
+// Please refer to their documentation on https://mochajs.org/ for help.
+//
+
+// The module 'assert' provides assertion methods from node
+import * as assert from 'assert';
+
+// You can import and use all API from the 'vscode' module
```

Comment:
Probably don't want this generic template text either

Reply:
@nklincoln I have updated files per your comments.  can you review it again please?